pkgs/build-support/coq/meta-fetch/default.nix

```diff
@@ -79,6 +79,10 @@ let
           cond = pr == null && (match "^github.*" domain) != null && artifact != null;
           out = "https://github.com/${owner}/${repo}/releases/download/${rev}/${artifact}";
         }
+        {
+          cond = pr == null && (match "^github.*" domain) != null && ext == "zip";
+          out = "https://${domain}/${owner}/${repo}/archive/refs/tags/${rev}.${ext}";
+        }
```

Comment:
For some reason the file given by https://github.com/MetaRocq/metarocq/archive/v1.4-9.1.zip is corrupted while the one at https://github.com/MetaRocq/metarocq/archive/refs/tags/v1.4-9.1.zip isn't. Not sure the above is the best hack ever but this seems to work.

Reply:
The first URL is wrong because “v1.4-9.1” is ambiguous: it is the name of a branch as well as the name of a tag.

The patch is wrong: there is no reason to believe that “ext == zip” means that “rev” is a tag name.